docs/content/docs/dev/testing.md

```diff
@@ -67,12 +67,220 @@ and an `admin` account with the username `admin` and passwort `admin` (which you
 
 Please change `bind_addresses` of `listeners` from `['::1', '127.0.0.1']` to `['0.0.0.0']` (in the `homeserver.yaml`), that means any address and allows remote connection (non-localhost).
 
+To avoid the change of server URL under VMWare, you can use NAT mode not Bridged mode as network.
+
+#### Ubuntu VM Guide under sqlite
+
+```shell
+sudo apt update
+sudo apt upgrade
+
+sudo apt install lsb-release wget apt-transport-https
+
+sudo wget -qO /usr/share/keyrings/matrix-org-archive-keyring.gpg https://packages.matrix.org/debian/matrix-org-archive-keyring.gpg
+
+sudo echo "deb [signed-by=/usr/share/keyrings/matrix-org-archive-keyring.gpg] https://packages.matrix.org/debian/ $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/matrix-org.list
+
+sudo apt update
+sudo apt upgrade
+sudo apt install matrix-synapse-py3
+```
+
+At the end of `sudo apt install matrix-synapse-py3`, you will get the following dialog.
+
+![Ubuntu ServerName](../../../static/images/ubuntu-servername.png)
+
+You have to enter `ds9.effektio.org` in this dialog, that is domain applied to all users in `effektio-test`.
+`server_name` in `/etc/matrix-synapse/homeserver.yaml` seems to not affect synapse config and the setting of this dialog during installation affects synapse config clearly.
+
+In `homeserver.yaml`, you have to change `bind_addresses: ['::1', '127.0.0.1']` to `bind_addresses: ['0.0.0.0']`.
+And append the following content to `homeserver.yaml`.
+
+```yaml
+allow_guest_access: true
+enable_registration_without_verification: true
+enable_registration: true
+registration_shared_secret: "2lyjkU7Ybp24rWR1TBJkut65RFcXZZA"
+
+rc_message:
+  per_second: 1000
+  burst_count: 1000
+
+rc_registration:
+  per_second: 1000
+  burst_count: 1000
+
+rc_login:
+  address:
+    per_second: 1000
+    burst_count: 1000
+```
```

Comment:
this need the extended set of limits, too, doesn't?

Reply:
Fixed